index.html

```diff
@@ -40,36 +42,272 @@ <h2 id="jobTitle">I am a Software Developer</h2>
+      <path d="M531.09 89.194L493.291 92.4748L500.434 51.6148L538.53 48.9305L575.138 46.2463L567.995 85.9133L560.554 125.282L523.946 129.159L531.09 89.194Z" fill="#FF6B00"/>
+      <path d="M319.775 150.931L363.526 146.159L356.681 188.809L398.943 183.44L391.8 224.897L349.538 230.862L305.787 237.125L312.93 194.177L267.393 199.844L220.07 205.809L226.916 160.774L274.238 155.703L281.381 111.264L326.918 107.387L319.775 150.931Z" fill="#FF6B00"/>
+      <path d="M233.761 115.44L240.904 69.8079L288.226 66.5272L281.381 111.264L233.761 115.44Z" fill="#262626"/>
+      <path d="M253.702 286.933L260.548 243.687L305.787 237.125L298.941 279.775L253.702 286.933Z" fill="#262626"/>
+      <path d="M439.718 178.37L479.005 173.598L471.862 213.564L464.719 253.231L425.432 259.494L384.955 266.055L391.8 224.897L432.575 219.23L439.718 178.37Z" fill="#262626"/>
+      <path d="M546.566 202.827L509.66 208.195L516.803 168.826L553.411 164.054L546.566 202.827Z" fill="#262626"/>
+      <path d="M596.269 121.405L589.126 159.879L553.411 164.054L560.554 125.282L596.269 121.405Z" fill="#262626"/>
+      <path d="M288.227 66.5272L295.37 21.4917L341.204 19.1057L334.061 63.2465L288.227 66.5272Z" fill="#262626"/>
+      </g>
+      </g>
+      <path d="M581.883 221.219L555.097 248.478L600.003 292.123C600.003 291.965 624.425 262.186 581.883 221.219Z" fill="#262626"/>
+      <path d="M568.017 210.347L637.345 128.099C637.345 128.099 663.973 156.618 669.961 173.95C677.996 197.27 666.809 211.293 666.809 211.293L600.002 291.965C600.002 291.965 611.505 283.299 598.742 249.738C586.61 217.753 568.017 210.347 568.017 210.347Z" fill="white" stroke="#262626" stroke-width="0.787818" stroke-miterlimit="10"/>
+      <path d="M347.743 100.053C347.743 100.053 309.14 78.4667 279.991 92.1747L341.283 42.0695L388.395 51.6809L347.743 100.053Z" fill="#262626"/>
+      <path d="M449.528 72.952L355.148 0.472717L292.753 55.4624L387.291 132.038L449.528 72.952Z" fill="white" stroke="#262626" stroke-width="0.787818" stroke-miterlimit="10"/>
+      <path d="M88.8671 280.778L107.145 334.823H20.327L0.473961 280.778H88.8671Z" fill="white" stroke="#262626" stroke-width="0.787818" stroke-miterlimit="10"/>
+      <path d="M154.255 280.778C154.255 280.778 157.564 276.997 148.74 260.138C139.917 243.121 115.494 220.431 115.494 220.431C115.494 220.431 105.568 210.505 103.835 208.772L175.684 260.61L154.255 280.778Z" fill="#262626"/>
+      <circle r="11.625" transform="matrix(-1 0 0 1 53.2695 309)" fill="white" stroke="#262626" stroke-width="0.75"/>
+      </g>
+      <defs>
+      <clipPath id="clip0_34_1555">
+      <rect width="673.269" height="353.1" fill="white" transform="matrix(-1 0 0 1 673.27 0)"/>
+      </clipPath>
+      <clipPath id="clip1_34_1555">
+      <rect width="376.199" height="317.933" fill="white" transform="matrix(1 0 0 -1 220.07 286.933)"/>
+      </clipPath>
+      </defs>
+      </svg>
```

Comment:
- [ ] As instructed by the previous reviewer, kindly make this a proper background image by adding from the stylesheet.
### Tip
Extract the SVG into its own file (eg ```about.svg```) and then import it as a ```background-image``` in your stylesheet